src/evaluate/load.py

```diff
@@ -73,7 +74,9 @@ def init_dynamic_modules(
     return dynamic_modules_path
 
 
-def import_main_class(module_path, dataset=True) -> Optional[Union[Type[DatasetBuilder], Type[Metric]]]:
+def import_main_class(
+    module_path, dataset=True, comparison=False
```

Comment:
We don't need `dataset` anymore - that's a remnant. Since we will maybe add oder types as well (e.g. "measurement" the following would be easier extendable.

```suggestion
    module_path, class_type="metric"
```